### DISCUSSION TITLE
newbie help

### DISCUSSION THREAD

**Discussion by np4001** [Esri/arcgis-cookbook#340] [Q&A] [answered]
Total newbie question(s). 
I have downloaded the cookbook and need to run it for enterprise. I have seen the tutorials but we do not have CINC we use chef automate with VS. Which files do I need to edit to set up the parameters? Do I edit the policyfile.rb with the needed information for these type settings?
                             node['arcgis']['version'] = ArcGIS version. Default value is 11.0.
                             node['arcgis']['run_as_user'] = Account used to run ArcGIS Server, Portal for ArcGIS, and ArcGIS Data Store. Default 
                                account name is arcgis.
                             node['arcgis']['run_as_password'] = Password for the account used to run ArcGIS Server, Portal for ArcGIS, and 
                                 ArcGIS Data Store. Default value isPa$$w0rdPa$$w0rd.

Or is there a reference that you can point me to for using this without CINC @pbobov ?

Thanks in advance!

**Comment by pbobov** [comment, marked as answer]
Hi @np4001,

If you have not used Chef before, I'd recommend getting started with [Deploy Base ArcGIS Enterprise on Single Machine](https://esri.github.io/arcgis-cookbook/tutorials/arcgis-enterprise-base/4.0.0.html).

You can use chef-client in place of cinc-client. Chef Cookbooks for ArcGIS support Cinc Client as well as Chef Infra Client. [CINC](https://cinc.sh/) is a Free-as-in-Beer distribution of the open source software of Chef Software Inc. 

Once you are comfortable with single machine and Chef in "solo" mode,  move to multi-machine deployments and possibly add Chef Server to the mix.

I have not tried using Chef Automate with Visual Studio.

Regards,
Pavel
